config/webpack.config.dev.js

```diff
@@ -119,8 +119,18 @@ module.exports = {
       // allow it implicitly so we also enable it.
       {
         test: /\.json$/,
+        exclude: /\/manifest.json$/,
         loader: 'json'
       },
+      // A special case for manifest.json to place it into build root directory.
+      {
+        test: /\/manifest.json$/,
+        include: [paths.appSrc],
```

Comment:
One thing I’m not sure about is whether it should be in `src`. I’m leaning towards putting it on the top level because you can’t `import` it from code now—you’ll get a string. What do you think?